src/services/message.service.ts

```diff
@@ -40,6 +43,52 @@ export class MessageService {
     }
   }
 
+  async reactionMessage(
```

Comment:
Not a good name
maybe `sendReactiveMessage` since it reacts to users doing something with a callback